<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeneral">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="actionCapitulos"/>
    <addaction name="actionNuevaPersona"/>
    <addaction name="actionNuevaCasa"/>
    <addaction name="actionNuevoCapitulo"/>
    <addaction name="actionNuevaProvincia"/>
    <addaction name="actionNuevoTema"/>
    <addaction name="actionNuevaObra"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuGeneral"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCapitulos">
   <property name="text">
    <string>Capítulos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionNuevoCapitulo">
   <property name="text">
    <string>Nuevo capítulo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNuevaCasa">
   <property name="text">
    <string>Nueva casa</string>
   </property>
  </action>
  <action name="actionNuevaPersona">
   <property name="text">
    <string>Nueva persona</string>
   </property>
  </action>
  <action name="actionNuevaObra">
   <property name="text">
    <string>Nueva obra</string>
   </property>
  </action>
  <action name="actionNuevaProvincia">
   <property name="text">
    <string>Nueva provincia</string>
   </property>
  </action>
  <action name="actionNuevoTema">
   <property name="text">
    <string>Nuevo &amp;tema</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
